README.md

```diff
@@ -1 +1,148 @@
-# GIS-DAY
\ No newline at end of file
+<a name="readme-top"></a>
+
+![](/images/screenshot.png)
+
```

Comment:
- [optional] I think you can use a smaller image (logo) instead of this large image. Perhaps your README file will look much more better. 💁‍♂️